### ISSUE TITLE
identify loss

### ISSUE THREAD

**Issue by DamDam123** [YuYin1/DA-GAN#1]
Hello, I want to ask when using lightcnn to calculate the identity loss, first use common.MeanShift to process the image, the result is a three-channel image, but lightcnn is to receive a single-channel image. I want to ask whether lightcnn is trained by you again?Or how to process the three-channel image into a single-channel image?Thank you very much!

**Comment by YuYin1** [comment]
I retrained the lightcnn with their provided training code.

**Comment by DamDam123** [comment]
Ok, thank you for your reply, and a small request, can you send me the pre-processed CAS-PEAL-R1 data set in your paper, hope to reply, thank you very much!

**Comment by DamDam123** [comment]
> I retrained the lightcnn with their provided training code.

Hello, I saw the cropped CAS-PEAL-R1 data set you shared, thank you very much, but I found that there are only 479 frontal, is this the training set or the test set? And the pose folder is empty, hope to reply, thank you！

**Comment by YuYin1** [comment]
The download link is added to the `Data` section in ReadMe. The dataset was still uploading when you downloaded it yesterday. But now it's all set.

**Comment by fishman2008** [comment]
Do you know what is the input size of the tensor for your trained LightCNN? The weight assigned to the identity loss is 0.1, which is sort of small. Will a larger weight help preserve more identity information during the reconstruction?

**Comment by YuYin1** [comment]
The input size is 128*128*3. 

As for the weight of identity loss, we tried with 1, but it generates more artifacts on the super-resolved faces.

**Comment by fishman2008** [comment]
@YuYin1  Thanks for your reply. But it appears the model is accepting a 1-channel, other than the 3-channel. I use the provided weights and model file. 

	size mismatch for module.conv1.filter.weight: copying a param with shape torch.Size([96, 1, 5, 5]) from checkpoint, the shape in current model is torch.Size([96, 3, 5, 5]).

**Comment by YuYin1** [comment]
I retrained the lightcnn with their provided training code. When I retrain the model, I used 3 channel input.

**Comment by ananiask8** [comment]
Can you provide the retrained lightcnn model? Thank you!

**Comment by zjh-hub** [comment]
@DamDam123 弄好了吗

**Comment by hui-vigil** [comment]
@DamDam123 请问弄好了吗

**Comment by marcinkusz** [comment]
In case anyone is still viewing this in the year of our Lord 2026 - the retrained weights are in one of the initial commits and are deleted by commit `f5c0e73b7739fc9db9909d7e7944604cc21c7860`.
